### PR TITLE
Add end copyright year date

### PR DESCRIPTION
**Pull request by fgs-dbudwin**

### CODE REVIEW

File: _layouts/default.html [hunk 1]
```diff
@@ -53,7 +53,7 @@ <h1 class="site-name"><a href="{{ site.baseurl }}/">{{ site.name }}</a></h1>
       <div class="container">
         <footer class="footer">
           {% include svg-icons.html %}
-          <p>FoxGuard Solutions, Copyright 2015</p>
+          <p>FoxGuard Solutions, Copyright 2015-{{ site.time | date: "%Y" }}</p>
```
Comment (FGS-JSavage) [inline _layouts/default.html]: Should this copy PUMP's style?
capture

Comment (FGS-JSavage) [inline _layouts/default.html]: And only show the current year, that is.

**Comment by fgs-dbudwin** [comment]
My understanding for the distinction between a single year, vs. a range is that a single year is used for static content (like a printed book) whereas a range is used for content that changes (like a website).  I would suggest the PUMP needs to be corrected.

File: _layouts/default.html (hunk 1 above)
Comment (fgs-dbudwin) [inline _layouts/default.html]: My understanding for the distinction between a single year, vs. a range is that a single year is used for static content (like a printed book) whereas a range is used for content that changes (like a website). I would suggest PUMP needs to be corrected.

**Comment by FGS-JSavage** [comment]
👍 :shipit:

**Pull request merged**